Source/Util.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Find index of a sorted array, to insert an item https://stackoverflow.com/a/29018745
+ * @param {*} sortedArr Array must be already sorted
+ * @param {*} target target value
+ * @param {*} compareFn compare function applicable to target and sortedArr items
```

Comment:
```suggestion
 * @param {(first, second) => number} compareFn compare function applicable to target and sortedArr items
```
Function signatures are valid TypeScript types.